homeassistant/components/config/config_entries.py

```diff
@@ -6,6 +6,7 @@
 from homeassistant.exceptions import Unauthorized
 from homeassistant.helpers.data_entry_flow import (
     FlowManagerIndexView, FlowManagerResourceView)
+from homeassistant.generated.config_flows import FLOWS
 
 
```

Comment:
Might be iter `custom_component/**/manifest.json` to load `config_flow` here.

Reply:
In the future 👍